geomesa-tools/bin/geomesa-kafka

```diff
@@ -247,5 +247,5 @@ else
         shift 1
     fi
     CLASSPATH="$GEOMESA_CP:$ACCUMULO_CP:$HADOOP_CP"
-    java ${GEOMESA_OPTS} -cp ${CLASSPATH} org.locationtech.geomesa.tools.Runner "$@"
+    java ${GEOMESA_OPTS} -cp ${CLASSPATH} org.locationtech.geomesa.tools.kafka.KafkaRunner "$@"
```

Comment:
Agreed.  Let's keep "geomesa" (laster to be named "geomesa-accumulo"?), and simply add "geomesa-kafka" for now.